drivers/client_access_control/spec/requests/clients_spec.rb

```diff
@@ -160,12 +183,49 @@
       setup_access_control(user, can_search_own_clients, Collection.system_collection(:window_data_sources))
     end
 
-    it 'allows index' do
+    it 'allows index with no params' do
       sign_in user
       get clients_path
       expect(response).to render_template(:index)
     end
 
+    it 'handles legacy GET search request' do
+      sign_in user
+      get clients_path, params: { q: 'test' }
```

Comment:
Legacy path is here

Reply:
sorry, does something need to change with that?